fvm/src/kernel/mod.rs

```diff
@@ -229,7 +241,15 @@ pub trait ActorOps {
     fn balance_of(&self, actor_id: ActorID) -> Result<TokenAmount>;
 }
 
+// we are not delegating this trait as ambassador seems to not support feature flags
```

Comment:
You know.... can we just get rid of the feature flag? The same way we did for "upgrades"? I.e.:

1. Use the feature flag to decide whether or not to _bind_ the syscall.
2. Don't put this method behind a feature flag.